### ISSUE TITLE
How to deal with ids?

### ISSUE THREAD

**Issue by danoc93** [liamks/csc490-a3#4]
I am assuming that: Adding users allows us to input the id, but updating them doesn't allow us to change it.

Am I correct? Or are we supposed to allow updates to occur...

**Comment by liamks** [comment]
Correct, no updating id's on the update page (although I won't deduct marks if it's there).
